requirements.txt

```diff
@@ -1,2 +1,3 @@
 aiofiles==0.8.0
-covalent==0.177.0
+covalent>=0.177.0
```

Comment:
pin to 0.177.0 and make sure to run the pre-commit hooks

Reply:
My bad, pinned now.